### PR TITLE
Revert "VS PR Diffs"

### PR DESCRIPTION
**Pull request by CalvinAllen**

### CODE REVIEW

File: shared/ui/Stream/OpenPullRequests.tsx [hunk 1]
```diff
@@ -309,6 +309,9 @@ export const OpenPullRequests = React.memo((props: Props) => {
 			currentRepoObject: getProviderPullRequestRepoObject(state),
 			reposState: state.repos,
 			maximized: settings.maximized,
+			// VS will not use sidebar-diffs currently, uses old method of going directly to
+			// details view when selecting a PR
+			isVS: state.ide?.name?.toUpperCase() === "VS",
```
Comment (CalvinAllen) [inline shared/ui/Stream/OpenPullRequests.tsx:314]: @bcanzanella This is a Codemark from a PR diff view, in Visual Studio.  Where am I going.

**Comment by CalvinAllen** [comment]
But are the ranges correct? Lines 191-193

```
	const exit = async () => {
		await dispatch(clearCurrentPullRequest());
	};
```
								
[shared/ui/Stream/PullRequest.tsx](https://github.com/teamcodestream/codestream/blob/ece2adc6ed5871824fb952ce53e667ae2e13ffee/shared/ui/Stream/PullRequest.tsx#L191-L193) (Lines 191-193)

File: shared/ui/Stream/OpenPullRequests.tsx (hunk 1 above)
Comment (bcanzanella) [reply shared/ui/Stream/OpenPullRequests.tsx:314]: hmmm what am i looking at?

Comment (bcanzanella) [reply shared/ui/Stream/OpenPullRequests.tsx:314]: oh wait

File: shared/ui/Stream/PullRequest.tsx [hunk 2]
```diff
@@ -141,6 +141,7 @@ export const PullRequest = () => {
 			currentPullRequest: currentPullRequest,
 			currentPullRequestLastUpdated: providerPullRequestLastUpdated,
 			composeCodemarkActive: state.context.composeCodemarkActive,
+			isVsIde: state.ide.name === "VS",
```
Comment (CalvinAllen) [inline shared/ui/Stream/PullRequest.tsx:144]: Line 144, the actual changed line

File: shared/ui/Stream/OpenPullRequests.tsx (hunk 1 above)
Comment (CalvinAllen) [reply shared/ui/Stream/OpenPullRequests.tsx:314]: Heres a reply from GitHub - what happens?

Comment (CalvinAllen) [reply shared/ui/Stream/OpenPullRequests.tsx:314]: This is a closed PR - just found one I could comment on (from CS)

**Pull request merged**